README.md

```diff
@@ -10,148 +10,148 @@ First, ensure that you have Go Version 1.22 on you system. Clone this repository
 
-    ptcp [/path/to/output] [ID] -n [newpath/in/ID]
+    pt cp [/path/to/output] [ID] -n [newpath/in/ID]
 
 When changing a file name copied into the `ID` 
 
-    ptcp [/path/to/outputfile] [ID] -n [newpath/newFileName]
+    pt cp [/path/to/outputfile] [ID] -n [newpath/newFileName]
 
 When specificying a file or directory that is being copied from the ID
 
-    ptcp [ID] [/path/to/output] -n [path/to/file/or/directory]
+    pt cp [ID] [/path/to/output] -n [path/to/file/or/directory]
 
 To produce a tar/gzipped output or unpack a tar/gzipped in the pairtree structure run 
 
-    ptcp -a [/path/to/ID.tgz] [ID]
+    pt cp -a [/path/to/ID.tgz] [ID]
 
 or 
 
-    ptcp -a [ID] [/path/to/dest]
+    pt cp -a [ID] [/path/to/dest]
 
 This provides a way to archive an item from the pairtree and un-archive it again back into a pairtree structure, but it's not intended as a way to create archives within the pairtree structure. Only the entire object can be archived from the pairtree meaning the `-a` and `-n` flags should never be used together. When an object is archived, a `.tgz` file will be created and named after the Pairtree object. It will contain a folder that is named the object ID. Unless otherwise specific with the `-d` option, the `.x` pattern will be followed so as not to overwrite other existing `.tgz` files that are named the same. When unarchiving a file into the pairtree, the `.tgz` file should contain a folder named after the pairtree object. The contents of that folder will fully overwrite the contents in the pairtree object. 
 
-## ptmv
+## mv
 
-Ptmv is a mv-like tool that can move files in and out of the Pairtree structure. Ptmv operates similarly to ptcp except it is destructive, removing the "from" source and overwriting the "to" destination (so deleting the existing directory, if there is one). Ptmv only works on the directory/Pairtree object level and not at the level of files within the Pairtree object, so all sources and targets should represent directories instead of individual files. 
+Mv is a mv-like tool that can move files in and out of the Pairtree structure. Mv operates similarly to cp except it is destructive, removing the "from" source and overwriting the "to" destination (so deleting the existing directory, if there is one). Mv only works on the directory/Pairtree object level and not at the level of files within the Pairtree object, so all sources and targets should represent directories instead of individual files. 
```

Comment:
Same (regarding `Mv`), I'll stop mentioning this at every place, though... you get the point and can decide from there.